storage/policy.go

```diff
@@ -0,0 +1,60 @@
+package storage
+
+// Policies is an array of policies.
+//
+// swagger:ignore
+type Policies []Policy
+
+// Policy specifies an ORY Access Policy document.
+//
+// swagger:ignore
+type Policy struct {
```

Comment:
Since these are 1:1 copies of https://github.com/ory/keto/blob/master/engine/ladon/policy.go#L31 I think it would make sense to move these completely (delete the files ladon/policy etc).

Reply:
Regarding tests, I cant complete those tests unless swagger spec is correctly generated and the new sdk has the updated fields. So could you help me resolve that issue at the earliest?